EIPS/eip-7732.md

```diff
@@ -132,14 +150,16 @@ The `ExecutionPayloadHeader` object is changed to only track the minimum informa
 State transition logic is modified by: 
 
 - A new beacon state getter `get_ptc` returns the PTC members for a given slot. 
-- `get_attesting_indices` is modified to not return the beacon committee members that are included in the PTC.
 - `process_withdrawals` is modified as follows. Withdrawals are obtained directly from the beacon state instead of the execution payload. They are deducted from the beacon chain. The beacon state `latest_withdrawals_root` is updated with the HTR of this list. The next execution payload MUST include withdrawals matching the `state.latest_withdrawals_root`. 
-- `process_execution_payload` is removed from `process_block`. Instead a new function `process_execution_payload_header` is included, this function validates the `SignedExecutionPayloadHeader` included in the `BeaconBlockBody` and transfers the payment from the builder's balance to the proposer. 
+- `process_execution_payload` is removed from `process_block`. Instead a new function `process_execution_payload_header` is included, this function validates the `SignedExecutionPayloadHeader` included in the `BeaconBlockBody`, deducts the payment from the builder's balance and adds a `BuilderPendingPayment` object to the beacon state.
```

Comment:
I don’t think “deducts the payment from the builder’s balance” is entirely accurate, but maybe I’m taking it too literally. The builder’s balance is actually deducted when `process_withdrawals` is called. It might also be worth adding a few lines about the changes to `get_expected_withdrawals`